### PR TITLE
add the label and fix the font

### PR DESCRIPTION
**Pull request by gulshan81**
Targets #1829

### Description
add `No Contract` label and fix the font

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by P4NK4J — CHANGES_REQUESTED** [review]
@gulshan81 Please fix the CI checks and then assign for review

File: Modules/Project/Resources/views/index.blade.php [hunk 1]
```diff
@@ -51,7 +51,7 @@
                 <tr>
                     <th class="w-33p sticky-top">Client/Project Name</th>
                     <th class="sticky-top">Team Members</th>
-                    <th class="sticky-top">Status</th>
+                    <th class="sticky-top"> Tags </th>
```
Comment (P4NK4J) [inline Modules/Project/Resources/views/index.blade.php:54]:
```suggestion
                    <th class="sticky-top">Tags</th>
```

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Comment by P4NK4J** [comment]
@gulshan81 Minor suggestion, else looks good.

File: Modules/Project/Resources/views/index.blade.php [hunk 2]
```diff
@@ -87,7 +87,7 @@
                                 <td>
                                     
                                     @if(empty($project->projectContracts->first()->contract_file_path))
-                                        <span class="text-dark font-weight-bold">No Contract</span>
+                                        <span>No Contract</span>
```
Comment (GauravGusain98) [inline Modules/Project/Resources/views/index.blade.php:90]: As per the issue requirement we have to make it as a label/badge. You can take reference form this document to see how to make a text as badge

https://getbootstrap.com/docs/4.0/components/badge/#contextual-variations

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

File: resources/sass/app.scss [hunk 3]
```diff
@@ -137,14 +137,14 @@ body,
...
-		}
+		}	
 
 		span:nth-child(6) {
 			left: calc(50% - 2px);
 			top: 9px;
 		}
 	}
-}
+
```
Comment (GauravGusain98) [inline resources/sass/app.scss:147]: I think there is one extra bracket. Please check, due to this the integration testing is also getting failed

File: resources/sass/app.scss [hunk 4]
```diff
@@ -754,7 +754,7 @@ span.total-chart-count{
 }
 .choices__list--multiple .choices__item {
 	background-color: #E4E4E4;
-	border: #E4E4E4;
+	border:#E4E4E4;
```
Comment (GauravGusain98) [inline resources/sass/app.scss:757]: Why you have removed the space?

File: Modules/Project/Resources/views/index.blade.php [hunk 5]
```diff
@@ -87,7 +87,10 @@
                                 <td>
                                     
                                     @if(empty($project->projectContracts->first()->contract_file_path))
-                                        <span class="text-dark font-weight-bold">No Contract</span>
+                                         <span class="badge badge-light"style="border: 1px solid;
+                                         font-size: 80%;
+                                         padding: 6px 0.5rem;
+                                         border-radius: 0.15rem " >No Contract</span>
```
Comment (GauravGusain98) [inline Modules/Project/Resources/views/index.blade.php:93]: Don't use style tag inside the blade file

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

File: resources/sass/app.scss [hunk 6]
```diff
@@ -785,6 +786,17 @@ span.total-chart-count{
...
+	flex-wrap: nowrap !important;
+}
+
+.style {
+	border: 1px solid;
+	font-size: 80%;
+	padding: 6px 0.5rem;
+	border-radius: 0.15rem;
+}
+
```
Comment (GauravGusain98) [inline resources/sass/app.scss:799]: Why you have added these class when you are not using them anywhere in your code.

File: Modules/Project/Resources/views/index.blade.php (hunk 5 above)
Comment (GauravGusain98) [reply Modules/Project/Resources/views/index.blade.php:93]: Lets remove the style tag and use the bootstrap classes. All these styles are provided by bootstrap

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Left some comment. Please check.

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**